plugins/modules/zos_unarchive.py

```diff
@@ -146,7 +146,7 @@
     description:
       - A list of directories, files or data set names to extract from the
         archive.
-      - GDS relative names are supported C(e.g. USER.GDG(-1)).
+      - GDS relative names are supported C(e.g. USER.GDG(-1\)).
```

Comment:
This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

Reply:
fixed